mlsdbtool/main.cpp

```diff
@@ -1143,6 +1143,105 @@ QMap<QString, QVector<BoundingBox> > regionBoundingBoxes()
           << cbboxes.value(QLatin1String("India"));
     retn.insert(QLatin1String("Devel"), devel);
 
+    QVector<BoundingBox> easterneurope;
+    easterneurope << cbboxes.value(QLatin1String("Belarus"))
+          << cbboxes.value(QLatin1String("Bulgaria"))
+          << cbboxes.value(QLatin1String("Czech_Republic"))
+          << cbboxes.value(QLatin1String("Hungary"))
+          << cbboxes.value(QLatin1String("Moldova"))
+          << cbboxes.value(QLatin1String("Poland"))
+          << cbboxes.value(QLatin1String("Romania"))
+          << cbboxes.value(QLatin1String("Slovakia"))
+          << cbboxes.value(QLatin1String("Ukraine"));
+    retn.insert(QLatin1String("Eastern_Europe"), easterneurope);
+
+    QVector<BoundingBox> northerneurope;
+    northerneurope << cbboxes.value(QLatin1String("Denmark"))
+          << cbboxes.value(QLatin1String("Estonia"))
+          << cbboxes.value(QLatin1String("Finland"))
+          << cbboxes.value(QLatin1String("Iceland"))
+          << cbboxes.value(QLatin1String("Ireland"))
+          << cbboxes.value(QLatin1String("Latvia"))
+          << cbboxes.value(QLatin1String("Lithuania"))
+          << cbboxes.value(QLatin1String("Norway"))
+          << cbboxes.value(QLatin1String("Sweden"))
+          << cbboxes.value(QLatin1String("United_Kingdom"));
```

Comment:
Interesting having UK and Ireland here instead of Western Europe, but there are definitions going both ways so not wrong either.